src/dialog/youmu.c

```diff
@@ -264,7 +264,7 @@ DIALOG_TASK(youmu, Stage4PreBoss) {
 
 	FACE(kurumi, dissastisfied);
 	MSG(kurumi, "Of course not! Who do you take me for?!");
-	MSG(kurumi, "No WAY I’d want to associate myself with the people running this mansion!");
+	MSG(kurumi, "No WAY I’d want to associate myself with the idiot running this mansion!");
```

Comment:
Plural?

I still think something like dorks or dweebs would be more fitting here since she's insulting their fashion sense rather than intelligence.